### PR TITLE
121 - Features

### LINKED ISSUE #121: Features block and section
Create features block and section

Screenshot 2023-03-28 at 09 26 27

Examples:

- https://www.moleculardevices.com/products/microplate-readers/multi-mode-readers/spectramax-i3x-readers
- https://www.moleculardevices.com/products/microplate-readers/multi-mode-readers/spectramax-id3-id5-readers
- https://www.moleculardevices.com/products/microplate-readers/multi-mode-readers/spectramax-m-series-readers
- https://www.moleculardevices.com/products/microplate-readers/multi-mode-readers/spectramax-mini-reader
- https://www.moleculardevices.com/products/microplate-readers/multi-mode-readers/spectramax-paradigm-reader
- https://www.moleculardevices.com/products/microplate-readers/multi-mode-readers/flexstation-3-reader
- https://www.moleculardevices.com/products/flipr-penta-high-throughput-cellular-screening-system
- https://www.moleculardevices.com/service-support/lab-automation-solutions#features

### PR DESCRIPTION
**Pull request by andreituicu**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #121

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/
- After: https://121-features--moleculardevices--hlxsites.hlx.page/

- Before: https://main--moleculardevices--hlxsites.hlx.page/drafts/tuicu/features-section
- After: https://121-features--moleculardevices--hlxsites.hlx.page/drafts/tuicu/features-section

### CODE REVIEW

**Comment by paolomoz** [comment]
The H2 title is slightly mispositioned, here is how I made it look like the original

.features-section padding-top: 45px
h2 margin-bottom: 0
.default-content-wrapper margin-bottom: 27px

File: styles/styles.css [hunk 1]
```diff
@@ -477,6 +477,67 @@ nav {
...
+
+main .section.features-section .default-content-wrapper {
+  padding-top: 0;
+  padding-bottom: 0;
+  text-align: center;
+  margin-bottom: 30px;
+}
+
+main .section.features-section h2 {
+  margin-bottom: 0;
```
Comment (andreituicu) [inline styles/styles.css:497]: > h2 margin-bottom: 0

@paolomoz Fixed. Thank you! ✅

File: styles/styles.css [hunk 2]
```diff
@@ -477,6 +477,67 @@ nav {
...
+  background-position: left bottom, left bottom, right top, right top;
+  background-repeat: no-repeat;
+  background-color: #f2f2f2;
+}
+
+main .section.features-section .default-content-wrapper {
+  padding-top: 0;
+  padding-bottom: 0;
+  text-align: center;
+  margin-bottom: 30px;
```
Comment (andreituicu) [inline styles/styles.css:493]: > .default-content-wrapper margin-bottom: 27px

If you agree, I would keep this at `30px`. there are indeed small variations in some examples, but having 30px works for majority of cases and for the header. `3px` is a very small difference, so I went for the simplified version. 🙂

File: styles/styles.css [hunk 3]
```diff
@@ -477,6 +477,67 @@ nav {
   display: block;
 }
 
+/* Features Section */
+main .section.features-section {
+  padding-top: 45px;
```
Comment (andreituicu) [inline styles/styles.css:482]: > .features-section padding-top: 45px

@paolomoz: fixed. thank you! ✅

**Comment by andreituicu** [comment]
@mhaack 

> Also I would add the background images for this into the repo, not content. We will otherwise need to add this to every word doc for products, makes is unreadable. And we need two infos on the section the style & the background  images.

Moved the background images in the code. ✅

**Review by lydiapuric — APPROVED** [review]
+1

**Review by paolomoz — APPROVED** [review]

**Review by mhaack — APPROVED** [review]

**Pull request merged**